helpers/content.go

```diff
@@ -359,6 +429,15 @@ func (c *RenderingContext) getConfig() *Blackfriday {
 	return c.Config
 }
 
+func (c *RenderingContext) getMmarkConfig() *Mmark {
+	c.configInit.Do(func() {
+		if c.MmarkConfig == nil {
+			c.MmarkConfig = NewMmark()
```

Comment:
The sequence number doesn't really solve Hugo's problem -- as there will still be potential id conflicts when rendering pages on the home page (two pages with the same title in the same position), but it would probably be "good enough" for most people.